domains/homehosting.json

```
@@ -0,0 +1,12 @@
+{
+	"description": "Student hosting for SSH devices without open ports.",
+	"owner": {
+		"username": "GenerelSchwerz",
+		"email": "generelschwerz@gmail.com",
```

remove the trailing comma
```suggestion
		"email": "generelschwerz@gmail.com"
```